packages/lib/src/components/Dropin/components/DropinComponent.tsx

```diff
@@ -114,6 +114,7 @@ export class DropinComponent extends Component<DropinComponentProps, DropinCompo
     render(props, { elements, instantPaymentElements, status, activePaymentMethod, cachedPaymentMethods }) {
         const isLoading = status.type === 'loading';
         const isRedirecting = status.type === 'redirect';
```

Comment:
Shouldn't we also check for `storedPaymentElements` as done here? 
https://github.com/Adyen/adyen-web/blob/v6/packages/lib/src/components/Dropin/components/DropinComponent.tsx#L131

Reply:
If I am not mistaken they are part of the Component state. We set them here https://github.com/Adyen/adyen-web/blob/v6/packages/lib/src/components/Dropin/components/DropinComponent.tsx#L36

Reply:
The link you shared is from v6 😄, we put stored and regular payments together in v5: https://github.com/Adyen/adyen-web/blob/9902c457f47549623b744c2428a1f44b445cf96d/packages/lib/src/components/Dropin/components/DropinComponent.tsx#L32

Reply:
Ouch!  Then you are totally correct 😄